xbmc/cores/AudioEngine/Sinks/AESinkPULSE.cpp

```diff
@@ -207,7 +207,14 @@ static void StreamStateCallback(pa_stream *s, void *userdata)
 
 static void StreamRequestCallback(pa_stream *s, size_t length, void *userdata)
 {
-  pa_threaded_mainloop* m = static_cast<pa_threaded_mainloop*>(userdata);
+  CAESinkPULSE *p = static_cast<CAESinkPULSE*>(userdata);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types